### ISSUE TITLE
Parameter dropdown label 

### ISSUE THREAD

**Issue by MasterRO94** [beyondcode/invoker-community#139]
There is no ability to set parameter dropdown labels and the invoker chooses not the best field for that.

Here are some examples where without the ability to change label makes hard/impossible to select correct value.

Also, it would be cool if an accessor can be the lable.

![Invoker –  Dropdown value 1 2021-02-19 02-24-41](https://user-images.githubusercontent.com/7365389/108439670-67db9900-725a-11eb-90c3-3e2d309d4377.png)

![Invoker –  Dropdown value 2 2021-02-19 02-24-41](https://user-images.githubusercontent.com/7365389/108439684-7033d400-725a-11eb-88ec-5b9dbe3eda0c.png)

**Comment by mpociot** [comment]
Right. Those labels are currently auto-detected. I'll add the ability to customize the label, just like you can do with relation labels.
